lib/query.js

```diff
@@ -184,6 +184,46 @@ Query.prototype.search = function (text) {
 	return this;
 };
 
+/**
+ * Get distinct
+ *
+ * @param {String} field for distinct
+ * @param {Object} query - query to filter the results
+ * @see http://mongodb.github.io/node-mongodb-native/2.0/api/Collection.html#distinct
+ * @api public
+ */
+
+Query.prototype.distinct = function (field, query) {
+	let self = this;
+	this.where(query);
+	return this.collection.then(function (collection) {
+		return collection.distinct(field, self.query);
+	});
+};
+
+/**
+ * Aggregation query
+ *
+ * @param {String} pipeline aggregation pipeline
+ * @param {Object} options - Options to be passed to aggregation pipeline
+ * @see http://mongodb.github.io/node-mongodb-native/2.0/api/Collection.html#aggregate
+ * @api public
+ */
+
+Query.prototype.aggregate = function (pipeline) {
+	return this.collection.then(function (collection) {
```

Comment:
What indentation is missing? also npm test passed checking style guide defined, is there any thing extra i need to check?